ERP/src/components/inputfield/InputField.tsx

```diff
@@ -66,32 +72,38 @@ const InputField: React.FC<InputFieldProps> = ({
     useEffect(() => {
         const min = minStock ? parseInt(minStock) : SLIDER_CONFIG.STOCK.min;
         const max = maxStock ? parseInt(maxStock) : SLIDER_CONFIG.STOCK.max;
-        
+
         // 현재 슬라이더 값과 다를 때만 업데이트 (무한 루프 방지)
         if (stockSliderValues[0] !== min || stockSliderValues[1] !== max) {
             setStockSliderValues([min, max]);
         }
-    }, [minStock, maxStock, stockSliderValues]);
+    }, [minStock, maxStock]);
 
     // 엔터키 입력 시 검색 실행
     const handleKeyDown = (e: React.KeyboardEvent<HTMLInputElement>) => {
-        if (e.key === "Enter") {
+        if (e.key === 'Enter') {
             onSearch();
         }
     };
 
-    // 판매합계 슬라이더 값 변경 핸들러
+    // 판매합계 슬라이더 값 변경 핸들러 (드래그 중 미리보기만 업데이트)
     const handleSalesSliderChange = (value: number | number[]) => {
         const [min, max] = value as number[];
         setSalesSliderValues([min, max]);
+    };
+    const handleSalesSliderAfterChange = (value: number | number[]) => {
+        const [min, max] = value as number[];
```

Comment:
넘버인 리스트로 단언할 수 있다면, 위에 value에서 number은 지워도 되지 않나요? 의견 남겨주세요!

```suggestion
const handleSalesSliderAfterChange = (value: number[]) => {
        const [min, max] = value
```